plugins/org.eclipse.fordiac.ide.model.commands/src/org/eclipse/fordiac/ide/model/commands/change/MapCommunicationCommand.java

```diff
@@ -44,7 +45,7 @@ protected FBNetworkElement createTargetElement() {
 		comm.setPosition(EcoreUtil.copy(srcElement.getPosition()));
 		comm.setTypeEntry(srcElement.getTypeEntry());
 		comm.setInterface(srcElement.getType().getInterfaceList().copy());
-		transferFBParams(srcElement, comm);
+		MappingTargetCreator.transferFBParams(srcElement, comm);
```

Comment:
If we copy the interface, do we still need to transfer the FB params?

Reply:
From what I tested in the MappingTargetCreator no.